### ISSUE TITLE
Problem: Haven't released to maven central in a 1.5 years

### ISSUE THREAD

**Issue by trevorbernard** [zeromq/jzmq#399]
Reference: #396

**Comment by utgarda** [comment]
Yeah, that's true. Currently building it to put the jar to our private maven repo or to Git, hence #412

**Comment by paulvi** [comment]
So it is now 2.5 years without jar in Maven Central.

That means no 0MQ 4.x on Central at all
